das-cli/src/common/config/store.py

```
@@ -102,21 +103,41 @@ def set_path(self, new_file_path: str) -> None:
+
     def exists(self) -> bool:
-        return len(self.get_content().items()) > 0
+        return isinstance(self.get_content(), dict) and len(self.get_content()) > 0
 
     def rewind(self):
+        self._new_content = {}
         try:
-            with open(self._file_path, "r") as f:
-                self._content = json.load(f)
-        except (FileNotFoundError, json.JSONDecodeError):
+            with open(self._file_path, "r", encoding="utf-8") as f:
```

Ruff (0.16.0)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)